build/source/engine/read_attrb.f90

```diff
@@ -97,70 +122,177 @@ USE get_ixname_module,only:get_ixAttr,get_ixType  ! access function to find inde
  ! build filename
  infile = trim(SETNGS_PATH)//trim(LOCAL_ATTRIBUTES)
  ! open file
- call file_open(trim(infile),unt,err,cmessage)
+ mode=nf90_NoWrite
+ call nc_file_open(trim(infile), mode, ncid, err, cmessage)
  if(err/=0)then; message=trim(message)//trim(cmessage); return; endif
```

Comment:
The routine `netcdf_err` is being used within the routine `file_open` (this is where the string `cmessage` comes from). No change necessary.